lib/davinci_dtr_test_kit/payer_server_groups/payer_server_adaptive_response_validation_test.rb

```diff
@@ -24,6 +24,9 @@ class PayerAdaptiveFormQuestionnaireResponseTest < Inferno::Test
       This test may process multiple resources, labeling messages with the corresponding tested resources
       in the order that they were received.
     )
+    verifies_requirements 'hl7.fhir.us.davinci-dtr_2.0.1@169', 'hl7.fhir.us.davinci-dtr_2.0.1@294',
+                          'hl7.fhir.us.davinci-dtr_2.0.1@303', 'hl7.fhir.us.davinci-dtr_2.0.1@305',
```

Comment:
requirement 305 isn't tested yet - we never look for operation outcomes.  Remove from here and other tests.

Reply:
Addressed in [6653e49](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/59/commits/6653e495e069efcc7e8474e6becffb5493393339)